msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -435,6 +472,73 @@ A color to use with and as a background for your outline icons.
 
 The value must be a valid HTML color code starting with '#', for example `#4464ee`.
 
+## elementRelationshipSet
+
+**Optional** &ndash; Object
+
+Describes relationships among individual app components, including  `staticTabs`, `configurableTabs`, `composeExtensions`, and `bots`. Used to specify runtime dependencies to ensure that the app only launches from applicable Microsoft 365 hosts, such as Teams, Outlook, and the Microsoft 365 (Office) app. See [Specify runtime requirements in your app manifest](../../m365-apps/specify-runtime-requirements.md) for more info.
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+| `oneWayDependencies`| Array|||Array containing one or more unidirectional dependency relationships among app components (each represented by `oneWayDependency` object with *dependent* (`element`) and *depended on* (`dependsOn`) [`element` objects](#element-object))|
```

Comment:
> Array containing one or more unidirectional dependency relationships among app components
@erikadoyle, since we have already mentioned the data type as an array, can we simply start the sentence as "Contains one or more..."